packages/v4/src/content/extensions/quickstarts.md

```diff
@@ -0,0 +1,13 @@
+---
+id: Quickstarts
+section: extensions
+---
+
+## Coming soon
+Quickstarts is a project sponsored by UXD. The PatternFly team is currently working on incorporating 
```

Comment:
@abigaeljamie In case you have other ideas for the project name punctuation. Several examples of how it could be mid-sentence:
- I like the `QuickStarts` project!
- I like the `quick-starts` project!
- I like the `Quickstarts` project!
- I like the `quick starts` project!

Reply:
Thanks for the tag, @jschuler ! The standard we follow for products across UXD and CCS is "quick starts" (two words, all lowercase). This is similar to how we write all other learning experiences in lowercase, like "tour," "tutorial," etc.